sdk/cognitiveservices/Azure.OpenAI.Inference/src/cadl-location.yaml

```diff
@@ -0,0 +1,5 @@
+directory: specification/cognitiveservices/OpenAI.Inference
+additionalDirectories:
+  - specification/cognitiveservices/OpenAI.Authoring
```

Comment:
does this work? The generated code looks like only contains things from `OpenAI.Inference`

Reply:
there are a couple items that get referenced in this folder